rfc/mediachain-rfc-3.md

```diff
@@ -14,15 +14,22 @@ We will also outline a roadmap toward improved, `Future` Indexer generations. Fu
 
 ## Terminology
 
-Relevant Mediachain terminology background for this RFC. See RFC-1[2] and RFC-2[3] for more information.
+Relevant Mediachain terminology background for this RFC. See also RFC-1[2] and RFC-2[3] for more information.
 
 - `Indexer` - Collection of Mediachain subsystems described in this RFC.
 - `Datastore` - Persistent content-addressed storage system, used to store Mediachain media and blocks. Currently IPFS[4].
 - `Journal` - Flattened and partially-reconciled view of the Mediachain knowledge base. Maintained by `Transactors`.
 - `Folding` - Producing a single flattened representation of a chain of updates, reconciling any conflicts and discarding bad information in the process.
 - `Clients` - Embedded API or external nodes, which are the interface for end-user nodes to receive a stream of updates to the `Journal` from `Transactor` nodes, and for communicating writes to the `Transactors`. They also conduct reads and writes to the `Datastore`.
 - `Transactors` - Core of Mediachain. Accepts writes to the `Journal` from `Clients`, serves `Clients` with views of the `Journal`, and records transactions on their behalf. (Referred to as `Peer Nodes` in RFC-2.)
-- `Media Identifier` - One of several types of strings that can be used to indicate the identity of a media work. Most Indexer subsystems will resolve this identifier into a canonical ID. See more in the `Media Identifiers` section.
+- `Media Identifier` - String used to identify a media work. The string's prefix determines which, of several possible identifier types, each string is. See the `Media Identifiers` section in this RFC.
```

Comment:
So the idea with the `Media Identifiers` is to have a name for all types of image identifiers which can be accepted by the Indexer APIs. In other words, `canonical IDs` are a subset of the possible identifier types the Indexer API can accept. Also currently supported is querying by image content e.g. data URIs of small images, or could even extend it to querying by image descriptors / vectors that represent abstract visual concepts instead of a particular image's content - as is sometimes done in some image QA systems.